packages/info/README.md

```diff
@@ -9,22 +9,55 @@ This package returns a set of information related to the local environment.
 ## Installation
 
 ```bash
+#npm
 npm i -D @webpack-cli/info
+
+#yarn
+yarn add @webpack-cli/info -D
 ```
 
 ## Usage
 
+### Args / Flags
+
+#### Output format
+
+| Flag         | Description                   | Type        |     |     |
```

Comment:
Output flags should be `--output-json`, `--output-markdown`

Reply:
Sounds great :+1: 
Done :rocket: